### PR TITLE
D10 1038 s41591 020 1061 7

### PR DESCRIPTION
**Pull request by lauradmartens**
Dataloader for "Characteristics of anti-CD19 CAR T cell infusion products associated with efficacy and toxicity in patients with large B cell lymphomas"

Paper: https://www.nature.com/articles/s41591-020-1061-7#Sec32
Data: https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE151511&format=file

### CODE REVIEW

File: sfaira/data/dataloaders/loaders/d10_1038_s41591_020_1061_7/human_blood_2020_10xsequencing_deng.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    assay_sc_obs_key:
+    assay_differentiation:
+    assay_differentiation_obs_key: 
+    assay_type_differentiation:
+    assay_type_differentiation_obs_key:
+    bio_sample:
+    bio_sample_obs_key: 
+    cell_line:
+    cell_line_obs_key:
+    cell_type:
```
Comment (davidsebfischer) [inline sfaira/data/dataloaders/loaders/d10_1038_s41591_020_1061_7/human_blood_2020_10xsequencing_deng.yaml:50]: This is T cell for all cells right? We can add this under `cell_type:`

Comment (lauradmartens) [reply sfaira/data/dataloaders/loaders/d10_1038_s41591_020_1061_7/human_blood_2020_10xsequencing_deng.yaml:50]: yes, it's CAR T cells for all cells, so just the coarsest ontology?

Comment (davidsebfischer) [reply sfaira/data/dataloaders/loaders/d10_1038_s41591_020_1061_7/human_blood_2020_10xsequencing_deng.yaml:50]: Yes, unless they specify that they only use Cd8 or Cd4 or so.

**Comment by davidsebfischer** [comment]
- [ ] move meta data maps into tsv

I ll do that.

**Pull request merged**